octoprint_mrbeam/wizard_config.py

```diff
@@ -0,0 +1,149 @@
+from flask.ext.babel import gettext
+from octoprint_mrbeam.mrb_logger import mrb_logger
+
+
+class WizardConfig:
+	def __init__(self, plugin):
```

Comment:
Could you please add a comment that explains where the order of the wizard-tabs is defined. Just to help our future selfs. thx

Reply:
Good idea